src/wp-admin/import.php

```diff
@@ -30,7 +30,7 @@
 get_current_screen()->set_help_sidebar(
 	'<p><strong>' . __( 'For more information:' ) . '</strong></p>' .
 	'<p>' . __( '<a href="https://wordpress.org/documentation/article/tools-import-screen/">Documentation on Import</a>' ) . '</p>' .
-	'<p>' . __( '<a href="https://wordpress.org/support/forums">Support</a>' ) . '</p>'
+	'<p>' . __( '<a href="https://wordpress.org/support/forums">Support forums</a>' ) . '</p>'
```

Comment:
```suggestion
	'<p>' . __( '<a href="https://wordpress.org/support/forums/">Support forums</a>' ) . '</p>'
```

Add the trailing slash to make it identical to other Support forums links.